### PR TITLE
[coq] Adapt to Coq's #402

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

**Comment by ejgallego** [comment]
Sorry for the no-description of the pull request; this is a rather trivial patch adapting to some representation changes in the Coq AST.

**Comment by maximedenes** [comment]
Yes, sorry, I should have added a description.

**Comment by JasonGross** [comment]
I will merge this once [travis is okay with the trunk build](https://travis-ci.org/mit-plv/bedrock/jobs/235949273).

File: Bedrock/reification/reif_tactics.ml.trunk [hunk 1]
```diff
@@ -29,17 +29,16 @@ let debug s = if debug then Format.printf s else Format.ifprintf (Format.std_for
...
-let dummy_loc = Loc.ghost
 (* Calling a global tactic *)
 let ltac_call tac (args:Tacexpr.glob_tactic_arg list) =
-  Tacexpr.TacArg(dummy_loc,Tacexpr.TacCall(dummy_loc, Misctypes.ArgArg(dummy_loc, Lazy.force tac),args))
+  Tacexpr.TacArg(Loc.tag @@ Tacexpr.TacCall(Loc.tag @@ (Misctypes.ArgArg(Loc.tag @@ Lazy.force tac),args)))
 
 (* Calling a locally bound tactic *)
 let ltac_lcall tac args =
-  Tacexpr.TacArg(dummy_loc,Tacexpr.TacCall(dummy_loc, Misctypes.ArgVar(dummy_loc, Names.id_of_string tac),args))
+  Tacexpr.TacArg(Loc.tag @@ Tacexpr.TacCall(Loc.tag @@ (Misctypes.ArgVar(Loc.tag @@ Names.id_of_string tac),args)))
```
Comment (SkySkimmer) [inline Bedrock/reification/reif_tactics.ml.trunk]: Is the added parenthesis necessary with the `@@`?

Comment (ejgallego) [inline Bedrock/reification/reif_tactics.ml.trunk]: @SkySkimmer I think so.

**Comment by ejgallego** [comment]
Thanks @JasonGross

**Pull request merged**